acme_diags/metrics/__init__.py

```diff
@@ -1,14 +1,47 @@
-from . import corr
-from . import rmse
-from . import max_cdms
-from . import mean
-from . import min_cdms
-from . import std
-
-corr = corr.CORR()
-rmse = rmse.RMSE()
-max_cdms = max_cdms.Max()
-mean = mean.Mean()
-min_cdms = min_cdms.Min()
-std = std.STD()
+import numpy
```

Comment:
I like it much better to have metrics calculation as functions in one file as modified.

Reply:
@chengzhuzhang, so do you like it the new way, or the older way where we had separate files?

Reply:
Like the new way:)